functional-test/src/test/groovy/org/rundeck/util/gui/pages/BasePage.groovy

```diff
@@ -126,11 +126,6 @@ abstract class BasePage {
                 .until(ExpectedConditions.elementToBeClickable(locator))
     }
 
-    WebElement waitForPresenceOfElementLocated(By locator) {
```

Comment:
why was this removed?

Reply:
That method is not used in the project, neither in oss and pro.